R/pkg/R/functions.R

```diff
@@ -2871,10 +2874,10 @@ setMethod("ifelse",
 
 #' @details
 #' \code{cume_dist}: Returns the cumulative distribution of values within a window partition,
-#' i.e. the fraction of rows that are below the current row.
-#' N = total number of rows in the partition
-#' cume_dist(x) = number of values before (and including) x / N
+#' i.e. the fraction of rows that are below the current row:
+#' number of values before (and including) x / total number of rows in the partition.
```

Comment:
with that many words it seems the fact this is a formula is lost. how about this like the other formula
```

(number of values before and including x) / (total number of rows in the partition)
```